client_taxi_driver/shared/src/commonMain/kotlin/di/GatewayModule.kt

```diff
@@ -0,0 +1,16 @@
+package di
+
+import data.remote.gateway.IdentityRemoteGateway
+import data.remote.fakegateway.IdentityFakeGateway
+import domain.gateway.remote.IIdentityRemoteGateway
+import domain.gateway.local.ILocalConfigurationGateway
+import data.local.gateway.LocalConfigurationGateway
+import org.koin.core.module.dsl.bind
+import org.koin.core.module.dsl.singleOf
+import org.koin.dsl.module
+
+val GatewayModule = module {
+//    singleOf(::IdentityRemoteGateway) { bind<IIdentityRemoteGateway>() }
```

Comment:
don't leave dead code

Reply:
this is not dead code its the remote module